### DISCUSSION TITLE
How do I hide the annotation completely

### DISCUSSION THREAD

**Discussion by DEDROX2K** [google/model-viewer#4877] [Q&A] [answered]
For some reason the buttons are still in vanilla style and it partially fades away, I want it to hide it.
![{F66FCB89-0D05-43AC-9A97-3A3348A3815E}](https://github.com/user-attachments/assets/eac7e7b7-5d34-446d-ba83-fc67605daa84)
Thanks.

**Comment by danilokrulj** [comment, marked as answer]
I think all of that can be changed through CSS. I just inspected some code with annotations and when it is faded away it gets this CSS style:

.annotation-wrapper.hide ::slotted(*) {
    opacity: var(--min-hotspot-opacity, 0.25);
}

Just change that value to 0 in your CSS and you should be good.

**Reply by DEDROX2K** [reply]
Well, tries that too and for some reason its not working. I think somewhere JS is interfering :(

.annotation-wrapper.hide ::slotted(*) {
    opacity: var(--min-hotspot-opacity, 0) !important;
    transition: opacity 13s !important;
}
